beacon-chain/core/blocks/block_operations_test.go

```diff
@@ -44,6 +45,21 @@ func setupInitialDeposits(t *testing.T, numDeposits int) ([]*pb.Deposit, []*bls.
 	return deposits, privKeys
 }
 
+func createAggregateSignature(t *testing.T, beaconState *pb.BeaconState, att *pb.Attestation, privKey *bls.SecretKey) []byte {
```

Comment:
this function looks like it would be useful for future tests also. maybe you could put this in helpers under a file `testHelpers.go`, we could then use it from there

Reply:
I think I'll just add a create sig function in the attestations.go like I did for the block signature in blocks.go